config/default_records/submission_templates/011_scrna_core_cdna_prep_submission_templates.wip.yml

```diff
@@ -16,9 +16,9 @@ Limber-Htp - scRNA Core cDNA Prep GEM-X 5p:
     request_type_keys: ["limber_scrna_core_cdna_prep_gem_x_5p"]
     product_line_name: Short Read
     product_catalogue_name: scRNA Core
-Limber-Htp - scRNA Core cDNA Prep Input:
+Limber-Htp - scRNA Core cDNA Prep GEM-X 5p PBMC Pools Input:
   submission_class_name: "LinearSubmission"
   related_records:
-    request_type_keys: ["limber_scrna_core_cdna_prep_input"]
+    request_type_keys: ["limber_scrna_core_cdna_prep_gem_x_5p_pbmc_pools_input"]
```

Comment:
I think you took the 'pbmc_pools' terminology out of the request type file, but forgot to update it here?

Reply:
So Abby asked specifically for the pbmc_pools_input to be included into the Submission template name so that it basically includes the plate purpose name, to avoid confusion with the other input submission template in the pipeline.
So I would suggest we also keep the pbmc_pools in the request type so they align. So this file is correct but the request type file above should include the pbmc_pools